documentation/sphinx/source/api-c.rst

```diff
@@ -481,7 +481,11 @@ An |database-blurb1| Modifications to a database are performed via transactions.
          |length-of| ``snapshot_command``
    
    .. note:: The function is exposing the functionality of the fdbcli command ``snapshot``. Please take a look at the documentation before using (see :ref:`disk-snapshot-backups`).
-         
+
+.. function:: double fdb_database_get_main_thread_busyness(FDBDatabase* database)
+
+   Returns a value between `0` and `1` representing the percentage of time the Network Thread is busy executing work from the TaskQueue. This value is updated every second.
```

Comment:
But there's a knob to control this? Or maybe this knob really should be a network option?

Reply:
This isn't necessarily something we want the client to be changing, so I don't think a network option is right for it.

Reply:
Why not? It doesn't seem harmful to change this at runtime

Reply:
Because ideally our calculation is good enough that it isn't something the client needs to worry about.

Reply:
I don't think there is a problem with clients changing this (and they can even if it's a knob), but it's more like something that we don't envision being something that is regularly changed like other network options.

The documentation of this function will also need to be less specific, as we are not committing to a particular implementation for busyness. In other words, it may end up being the case that we don't measure busyness by execution time of the network thread. Rather, this is an opaque value where 0 means we don't think the main thread is busy at all and 1 means we are completely busy. We could even end up using larger values to indicate the size of a backlog.

Reply:
I talked to A.J and Lukas and we think its better to keep it as a knob (as we don't generally expect people to change the monitor interval). I'll update this documentation to say the default it 1s

Reply:
I am fine with leaving it as a knob (as AJ pointed out a client can still change it). My comment meant more like "if a user can change it the documentation should be worded differently", so if you change this I am happy